### DISCUSSION TITLE
Custom Query Shortcuts with {value}

### DISCUSSION THREAD

**Discussion by k1n1mod** [Flow-Launcher/Flow.Launcher#2013] [Q&A] [answered]
Hello all,
I would like to make a "Custom Query Shortcuts" with variables. 
E.g. my link: https://www.youtube.com/watch?v={values}
If I create the shortcut:
Shortcut "yt=" 
Expansion "https://www.youtube.com/watch?v="
Later I would like to press [Alt] + [Space] and then only enter yt=dQw4w9WgXcQ and then the youtube video should be opened. How can I do this so that I can enter variables according to my parameter and that it is not searched in google but that my shortcut + the variable are called?
Thank you very much and I hope you could understand me.

**Comment by VictoriousRaptor** [comment, marked as answer]
You can combine plugin keyword, custom shortcut to achieve this. For example, set your your URL plugin keyword to "url" and created a shortcut "yt=" -> "url https://www.youtube.com/watch?v=". And type yt=dQw4w9WgXcQ to open your URL. To not search every query in google please adjust your plugin's keyword. Hope my explantion is useful enough.
